### DISCUSSION TITLE
"Restart to update" notification is given repeatedly to the user.

### DISCUSSION THREAD

**Discussion by Teapot4195** [ppy/osu#29774] [Q&A] [answered]
It appears the intended behaviour was introduced in a9250a0. However due to `scheduleRecheck` variable not being set to false before returning on line 70, the finally clause on 112 schedules a new check for 30 minutes.

Sent to the discussions from [here](https://discord.com/channels/188630481301012481/188630652340404224/1282108740568354989). Do we want to keep the current behaviour of bugging the user every 30 minutes or do we want to fix this and only notify the user once per game start?

**Comment by peppy** [comment, marked as answer]
See https://github.com/ppy/osu/pull/29787.
